pkg/kor/all.go

```diff
@@ -201,25 +201,27 @@ func GetUnusedAll(includeExcludeLists IncludeExcludeLists, filterOpts *FilterOpt
 		response[namespace] = resourceMap
 	}
 
-	var allDiffs []ResourceDiff
-	noNamespaceResourceMap := make(map[string][]string)
-	crdDiff := getUnusedCrds(apiExtClient, dynamicClient)
-	crdOutput := FormatOutputAll("", []ResourceDiff{crdDiff}, opts)
-	outputBuffer.WriteString(crdOutput)
-	outputBuffer.WriteString("\n")
-	noNamespaceResourceMap[crdDiff.resourceType] = crdDiff.diff
-
-	pvDiff := getUnusedPvs(clientset, filterOpts)
-	pvOutput := FormatOutputAll("", []ResourceDiff{pvDiff}, opts)
-	outputBuffer.WriteString(pvOutput)
-	outputBuffer.WriteString("\n")
-	noNamespaceResourceMap[pvDiff.resourceType] = pvDiff.diff
-
-	output := FormatOutputAll("", allDiffs, opts)
-
-	outputBuffer.WriteString(output)
-	outputBuffer.WriteString("\n")
-	response[""] = noNamespaceResourceMap
+	if opts.NoNamespaced {
```

Comment:
This should be true in a case that a person chooses all namespaces(or no namespace which is all)
The Readme should reflect this